Client/mods/deathmatch/logic/CClientVehicleManager.h

```diff
@@ -34,6 +34,7 @@ class CClientVehicleManager
 
     static bool               IsTrainModel(unsigned long ulModel);
     static bool               IsValidModel(unsigned long ulModel);
+    static bool               IsStandartModel(unsigned long ulModel);
```

Comment:
Typo? Is it supposed to be `IsStandardModel`?

Reply:
Yeah